src/components/RoomCard.tsx

```diff
@@ -68,7 +67,7 @@ export const RoomCard: React.FC<{
       setLoading(true);
 
       const res = await axios.request<PricedOfferResponse>(
-        new PricedOfferRequest(offer.id)
+        new PricedOfferRequest(offer.id as string)
```

Comment:
`as` this ts error was warning that Offer does not have id (I expect it says unknown)